build.sbt

```diff
@@ -1982,6 +1982,7 @@ lazy val designer = (project in file("designer/server"))
       (liteEmbeddedDeploymentManager / assembly).value
       (liteK8sDeploymentManager / assembly).value
       (defaultModel / assembly).value
+      (flinkBaseUnboundedComponents / assembly).value
```

Comment:
Is it related to this change, or it was fixed by the way?

Reply:
It is related - I make `ProcessingTypeDataProviderAccessRestrictionTest` use higher level abstractions, before change, it used `LocalProcessingTypeDataLoader` as well.